# Publish the desired target goal saved under the given name
string name
---
geometry_msgs/Pose goal